doc/base/classes.xml

```diff
@@ -2613,7 +2616,7 @@
 			<argument index="0" name="to_pos" type="Vector3">
 			</argument>
 			<description>
-				Returns the id of closest point of given point.  -1 is returned if there are no points on AStar.
+				Returns the id of the closest point to the given position. -1 is returned if there are no points in the points pool.
```

Comment:
Suggestion without passive voice: `Returns the closest point's id. Returns -1 if there are no points in the points pool.`